tools/model-compiler/src/main/java/io/spine/tools/gradle/compiler/Extension.java

```diff
@@ -354,6 +358,18 @@ public void generateInterfaces(Action<? super GeneratedInterfaces> action) {
         action.execute(generateInterfaces);
     }
 
+    @SuppressWarnings("unused")
+        // Used by Gradle to configure `generateMethods` with a closure.
+    public void generateMethods(Closure closure){
+        ConfigureUtil.configure(closure, generateMethods);
+    }
+
+    @SuppressWarnings("unused")
+    // Used by Gradle to configure `generateMethods` with a closure.
```

Comment:
Why is it indented in a different manner? See :362. Let's try to fit that into a single line.